src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -118,7 +123,6 @@ interface saveDocumentResponseItemDto {
     Etag: number;
     Method: string;
```

Comment:
What about this time? You removed usage of this typings in editDocument.ts. Why?